Practical Example/RWTakeHome/RWTakeHome/Extenstion/HTTPURLResponse+Ext.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  HTTPURLResponse+Ext.swift
+//  RWTakeHome
+//
+//  Created by Nick Nguyen on 10/7/20.
+//
+
+import UIKit
+
+extension HTTPURLResponse {
+  var isOK: Bool {
+    return statusCode == 200
+  }
```

Comment:
This is neat. One catch, usually everything 200 and up is considered a successful response. With your code, a server returning 204 no content would be considered invalid.